recipes/gunicorn/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "gunicorn" %}
+{% set version = "19.6.0" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version}}.tar.gz
+  url: https://github.com/benoitc/gunicorn/archive/{{ version }}.tar.gz
+  sha256: 1e0de4957bea60bfcff5215664bdfc3cf02c78e2aae9586766a9b4b437aebbb0
+
+build:
+  number: {{ build }}
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  preserve_egg_dir: yes
+  entry_points:
```

Comment:
Failing with:

```shell
Compiling /opt/conda/envs/_build/lib/python2.7/site-packages/gunicorn/workers/_gaiohttp.py ...
  File "/opt/conda/envs/_build/lib/python2.7/site-packages/gunicorn/workers/_gaiohttp.py", line 84
    yield from self.wsgi.close()
             ^
SyntaxError: invalid syntax
```

Maybe this is no longer Python 2 compat. If so you must add `skip: True  # [py2k]` here.